### PR TITLE
add create admin consumer

### PR DESCRIPTION
**Pull request by BelehovEgor**

### CODE REVIEW

File: src/UserService.Data/UserRepository.cs [hunk 1]
```diff
@@ -86,6 +86,25 @@ public Guid Create(DbUser dbUser, string password)
...
         }
 
+        public Guid Create(DbUser dbUser, DbUserCredentials credentials)
+        {
+            if (dbUser == null)
+            {
+                throw new ArgumentNullException(nameof(dbUser));
+            }
+
+            if(credentials == null)
```
Comment (TIove) [inline src/UserService.Data/UserRepository.cs:96]: Typo

File: src/UserService.Business/Commands/User/CreateUserCommand.cs [hunk 2]
```diff
@@ -111,7 +111,7 @@ private void SendEmail(DbUser dbUser, string password, List<string> errors)
 
             //TODO: fix add specific template language
             string templateLanguage = "en";
-            var senderId = _httpContextAccessor.HttpContext.GetUserId();
+            var senderId = Guid.NewGuid();//_httpContextAccessor.HttpContext.GetUserId();
```
Comment (TIove) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:114]: What is comment? For what?

File: src/UserService.Mappers/Db/DbUserMapper.cs [hunk 3]
```diff
@@ -69,5 +72,39 @@ public DbUser Map(CreateUserRequest request, Guid? avatarImageId)
...
+                Id = userId,
+                FirstName = request.FirstName,
+                LastName = request.LastName,
+                MiddleName = request.MiddleName,
+                Status = (int)UserStatus.WorkFromOffice,
+                AvatarFileId = null,
+                IsActive = true,
+                IsAdmin = true,
+                CreatedAt = DateTime.UtcNow,
+                StartWorkingAt = DateTime.UtcNow,
```
Comment (MariyaSin) [inline src/UserService.Mappers/Db/DbUserMapper.cs:96]: remove

**Review by TIove — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

File: src/UserService.Mappers/Db/DbUserMapper.cs (hunk 3 above)
Comment (BelehovEgor) [reply src/UserService.Mappers/Db/DbUserMapper.cs]: fixed

File: src/UserService.Data/UserRepository.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/UserService.Data/UserRepository.cs:96]: where?

Comment (BelehovEgor) [reply src/UserService.Data/UserRepository.cs]: fixed

**Review by djino97 — APPROVED** [review]

**Pull request merged**